### PR TITLE
objectives

### PR DESCRIPTION
**Pull request by SophieDeBenedetto**
needs review. adding this lesson before they do crowdfunding and library lab. without a walk through of group by, order by, having, etc. there is no way students will get through those labs. 

@jmburges @AnnJohn

### CODE REVIEW

**Comment by SophieDeBenedetto** [comment]
@AnnJohn @jmburges stopping work on this now to switch from hashes. I got really stuck on the multiple joins and grouping walk through. can't get my queries right, will not some help from anyone who is stronger in sql. i know this isn't a priority right now but just want to reiterate that there is no way students will be able to proceed through crowdfunding and library lab before we resolve this. just keep in mind for when we get to this unit.

File: README.md [hunk 1]
```diff
@@ -1 +1,184 @@
...
+SELECT column_name, column_name
+FROM table_name
+ORDER BY column_name ASC|DESC, column_name ASC|DESC;
+```
+
+`ORDER BY()` will automatically sort the returned values in ascending order. Use the `DESC` keyword, as above, to sort in descending order. 
+
+#### Exercise
+
+Imagine you're working for an important investment firm in Manhattan. The investors are interested in investing in a lucrative and popular cat. They need your help to decide which cat that will be. They want a list of famous and wealthy cats. We can do that with a basic SELECT statement:
```
Comment (jmburges) [inline README.md]: SELECT in backticks

**Comment by jmburges** [comment]
happy to help a bit more. whenever that time comes

File: README.md [hunk 2]
```diff
@@ -1 +1,184 @@
...
+FROM table_name
+WHERE column_name operator value
+GROUP BY column_name;
+```
+
+#### Exercise
+
+Let's calculate the sum of the net worth of all of the cats that belong to our second owner:
+
+```sql
```
Comment (jmburges) [inline README.md:174]: Did Sum without really introducing it. Also This query should be line by line instead of one big one, so hard to read

**Comment by jmburges** [comment]
:shipit: and then raise issue with what still needs to get done. Please don't forget to raise the issue

**Pull request merged**